src/seeds/users.seed.ts

```diff
@@ -0,0 +1,18 @@
+import { hash } from '@core/encryption/hash.js';
+import { UserDbDatasource } from '@data/users/user.db.datasource.js';
+import { configureSeeds } from './config.seed.js';
+
+const seedUsersDb = async () => {
+  for (let i = 0; i < 50; i++) {
+    await UserDbDatasource.create({
+      email: `user${i}@example.com`,
+      password: await hash('password123'),
+      name: `User ${i}`,
+      birthDate: '1990-01-01T00:00:00.000Z',
+    });
+  }
+};
+
+configureSeeds();
+
+seedUsersDb();
```

Comment:
```suggestion
await configureSeeds();
seedUsersDb();
```

Usando ESM, vc pode ter await na raiz do arquivo. E eu recomendo usar aqui pra garantir que a configuração será feita antes do seed.

Eu só tomaria cuidado com códigos na raiz de um arquivo, pq isso faz com que ele rode quando for importado, o que pode ocasionar em side-effects como rodar antes do que vc queria. Mas tomando esse cuidado, não vejo muito problema.